packages/core/src/types.ts

```diff
@@ -1288,6 +1290,7 @@ export interface ActorRef<TEvent extends EventObject, TEmitted = any>
   send: Sender<TEvent>; // TODO: this should just be TEvent
   id: string;
   getSnapshot: () => TEmitted | undefined;
+  start?: () => void;
```

Comment:
is this used anywhere? the `ActorRef` type represents already live actor - i don't think it should have a start method 🤔

Reply:
I thought it was used in v5, will double-check. One use-case that came to mind was for _restarting_ actors:

```js
someActor.start(snapshot);
```

But there might be a better way to do this.

Reply:
Since this is not used anywhere and there is no clear use case defined for it - I would just drop this from this type for the time being. We can always revisit.